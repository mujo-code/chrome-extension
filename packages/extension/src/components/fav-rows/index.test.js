import React from 'react'
import { create } from 'react-test-renderer'
import { FavRows } from '.'

const items = [
  {
    title: 'foo',
    url: 'https://foo.com',
  },
  {
    title: 'bar',
    url: 'https://bar.com',
    isUsed: true,
  },
]

test('FavRows component matches snapshot', () => {
  const tree = create(<FavRows items={items} />).toJSON()
  expect(tree).toMatchSnapshot()
})
