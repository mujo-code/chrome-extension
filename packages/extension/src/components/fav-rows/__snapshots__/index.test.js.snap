// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavRows component matches snapshot 1`] = `
<div
  className="css-gvoll6"
  flexWrap="wrap"
>
  <div
    className="css-s8fzq"
    height="xxl"
    maxHeight="xxl"
    width="xxl"
  >
    <a
      backgroundColor="outerSpace"
      className="css-1ioivri"
      disabled={true}
      href="https://foo.com"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <link
        href="https://foo.com"
        rel="preconnect"
      />
      <img
        alt="foo"
        className="css-1numroz"
        height="m"
        onError={[Function]}
        onLoad={[Function]}
        src="https://getmujo.com/api/icon?site=https%3A%2F%2Ffoo.com"
        width="m"
      />
      <div
        className="css-79elbk"
      >
        <div
          className="css-4pw2fe"
        >
          <span
            className="css-xrmt9z"
          >
            <div
              className="css-nxex8j"
            />
            <div
              className="css-91tplw"
            >
              foo
            </div>
          </span>
        </div>
      </div>
    </a>
  </div>
  <div
    className="css-s8fzq"
    height="xxl"
    maxHeight="xxl"
    width="xxl"
  >
    <a
      backgroundColor="outerSpace"
      className="css-1ioivri"
      disabled={true}
      href={null}
      onClick={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <link
        href="https://bar.com"
        rel="preconnect"
      />
      <img
        alt="bar"
        className="css-64d9dy"
        height="m"
        onError={[Function]}
        onLoad={[Function]}
        src="https://getmujo.com/api/icon?site=https%3A%2F%2Fbar.com"
        width="m"
      />
      <div
        className="css-79elbk"
      >
        <div
          className="css-4pw2fe"
        >
          <span
            className="css-xrmt9z"
          >
            <div
              className="css-nxex8j"
            />
            <div
              className="css-91tplw"
            >
              Bring back this link by taking a break
            </div>
          </span>
        </div>
      </div>
    </a>
  </div>
</div>
`;
